cli/src/semgrep/commands/ci.py

```diff
@@ -145,6 +181,19 @@ def fix_head_if_github_action(metadata: GitMeta) -> Iterator[None]:
     is_flag=True,
     hidden=True,
 )
+@click.option(
+    "--fail-open",
+    "fail_open",
+    type=FAIL_OPEN_STATE_TYPE,
+    help="""
+        Configures how scan status is sent to the fail-open server.
+        If 'on', fail-open are always sent.
+        If 'off', fail-open are disabled altogether and not sent.
+        If absent, the SEMGREP_SEND_FAIL_OPEN environment variable value will be used.
+        If no environment variable, defaults to 'off'.
+    """,
+    envvar="SEMGREP_SEND_FAIL_OPEN",
+)
```

Comment:
@raghavjain3 I thought we decided on a less jargon-y name for this option. Did we reverse that decision?

Reply:
I think we are going with `--no-suppress-errors` when we flip the default behavior, but not sure what we'd wanna call this.

Reply:
Yes. Renaming to `--no-suppress-errors` to match what is mentioned in the doc